frontend/src/features/admin-form/settings/components/MultiLanguageSection/TranslationListSection.tsx

```diff
@@ -345,9 +375,26 @@ export const TranslationListSection = ({
             )
           })}
 
+          {/* Form Logic Prevent Submission Message Translation */}
+          {formLogicPreventSubmissions && (
+            <>
+              <QuestionRow
+                key="Form Logic"
```

Comment:
Even though we are not rendering in a list manner, I think its required because there are other `<QuestionRow>` components being rendered at the same nested level as itself. Hence requiring a key to uniquely identify the `<QuestionRow>` components from each other. I keep seeing the error on my console regarding this, hence I added it. I am not very sure if its required, but when I add it, the error on console seems to disappear.